### ISSUE TITLE
is there a way to export this to a vector image of the network or export to DOT or Cytoscape format for manipulation?

### ISSUE THREAD

**Issue by cjieming** [datastorm-open/visNetwork#204]
is there a way to export this to a vector image of the network or export to DOT or Cytoscape format for manipulation?

**Comment by skanskan** [comment]
It would be great to be able to export it to svg or vectorial svg for publishing.

**Comment by bthieurmel** [comment]
Not at moment. And not so easy. Needed additionnal tools / library

**Comment by dockstreet** [comment]
Hmmm... I think this would perhaps help out what I'm trying to do as well, with this? 

https://github.com/datastorm-open/visNetwork/issues/208

I'm sure it's probably not easy to do, and not sure how to approach.... 
The slickR author looks like he is using "xmlSVG" perhaps and svglite and gdtools? I'm not sure if that would help at all because of the canvas?

Should this be reopened?

https://github.com/almende/vis/issues/1784

**Comment by anz000** [comment]
@cjieming Yes it can be exported to DOT. You need to hack into the vis.js code a bit, and get the nodes and edges information with xy positions, and other properities (size/color..etc).

**Comment by zx8754** [comment]
@anz000 if you have been through this path already, do you mind kindly share an example?

**Comment by anz000** [comment]
It's been a long time, but I can give you the directions to get a DOT output and convert it to svg (the final look will be different than the visNetwork one)

**Comment by cjieming** [comment]
That would be great! Thank you! Please let me know how to do that. Thanks!

**Comment by cjieming** [comment]
sorry I have been tied up in multiple projects but I will get back to trying this again and will let people know how this goes for me, esp shoutout to @anz000 for the example, and to all the rest for responding!
Thanks!!

**Comment by joshbiology** [comment]
Hi @cjieming, any updates on this feature? Thanks!
